.gitattributes

```diff
@@ -1,3 +1,93 @@
-# *.* eol=lf
+# Recipe from: https://help.github.com/articles/dealing-with-line-endings/#platform-all
+
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
+
+# Explicitly declare text files you want to always be normalized and converted
+# to native line endings on checkout.
+*.abnf text
+*.agc text
+*.asn text
+*.b text
+*.bas text
+*.bnf text
+*.c text
+*.cl text
+*.cpp text
+*.cs text
+*.css text
+*.csv text
+*.dgs text
+*.dot text
+*.erl text
+*.errors text
+*.faa text
+*.ffn text
+*.fna text
+*.g text
+*.g4 text
```

Comment:
Should be changed to eol=lf

Reply:
Since my commits did not touch a single `.g4` file, I do not see the need to enforce `lf` endings. 
The `*.g4 text` line (together with the `* text=auto` line) make sure that the line endings in the git repo are unified to `lf`. On the working tree they are converted to platform specific line endigs (and vice versa).